ts/features/itwallet/common/components/ItwCredentialClaim.tsx

```diff
@@ -270,14 +267,8 @@ const DrivingPrivilegesClaimItem = ({
   claim: DrivingPrivilegeClaimType;
   detailsButtonVisible?: boolean;
 }) => {
-  const localExpiryDate = localeDateFormat(
-    claim.expiry_date,
-    I18n.t("global.dateFormats.shortFormat")
-  );
-  const localIssueDate = localeDateFormat(
-    claim.issue_date,
-    I18n.t("global.dateFormats.shortFormat")
-  );
+  const localExpiryDate = claim.expiry_date.toString("DD/MM/YY");
+  const localIssueDate = claim.issue_date.toString("DD/MM/YY");
```

Comment:
According to the [Figma design](https://www.figma.com/design/pVD9XCba7crkg5yOIdJCXl/%5BIT-Wallet%5D---%E2%9C%85-Approved-only-%E2%9C%85?node-id=3693-20130&t=tpqgtpgGQwSZKUJc-4), these dates should follow the 'DD/MM/YYYY' format.

```suggestion
  const localExpiryDate = claim.expiry_date.toString("DD/MM/YYYY");
  const localIssueDate = claim.issue_date.toString("DD/MM/YYYY");
```